extensions/opentelemetry/opentelemetry-exporter-otlp/runtime/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-opentelemetry-exporter-otlp-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-opentelemetry-exporter-otlp</artifactId>
+    <name>Quarkus - OpenTelemetry Exporter - OTLP - Runtime</name>
+    <description>Enable OTLP Exporter for OpenTelemetry</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-opentelemetry</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-grpc-common</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-exporter-otlp</artifactId>
+            <exclusions>
```

Comment:
One other thing I just thought of.

For the moment, we should probably only depend on `opentelemetry-exporter-otlp-trace` as we're not supporting exports for anything other than tracing right now.

When we add other signal exporting in the future we can switch back to the full dependency.

Reply:
Yup, we can make this just the `oltp-trace`, but today the `exporter-otlp` only depends on the `oltp-trace`. But we can change it to make sure if the `all` pom changes we are just with the trace dependencies.

<img width="664" alt="image" src="https://user-images.githubusercontent.com/8834774/121037296-f413bc80-c774-11eb-8875-34e1e1b1ebe9.png">

Reply:
Right, probably best to explicitly depend on just the `trace` artifact as I expect at some point soon some of that will change, and don't want to bring it in until we're ready to handle it